src/encodings/tabularpreprocessing.jl

```diff
@@ -0,0 +1,117 @@
+"""
+    EncodedTableRow{M, N} <: Block
+
+Block for processed rows having a tuple of M categorical and 
+N continuous value collections.
+"""
+
+struct EncodedTableRow{M, N} <: Block
+    catcols
+    contcols
+    categorydict
+end
+
+function EncodedTableRow(catcols, contcols, categorydict)
+    EncodedTableRow{length(catcols), length(contcols)}(catcols, contcols, categorydict)
+end
+
+function checkblock(::EncodedTableRow{M, N}, x) where {M, N}
```

Comment:
Maybe we should restrain `x` to be a 2-tuple. Unless there is a reason not to?

Reply:
Yeah, I think we can do that.